XpringKit.podspec

```diff
@@ -14,7 +14,7 @@ Pod::Spec.new do |spec|
   spec.author       = { "Xpring Engineering" => "xpring@ripple.com" }
   spec.source       = { :git => "https://github.com/xpring-eng/XpringKit.git", :tag => "v3.1.1" }
 
-  spec.swift_versions = [4.2]
+  spec.swift_versions = [5.1]
```

Comment:
what does this bump mean?

Reply:
Ah yes, so the latest version of Swift is actually 5.2.  (Keefer suggested 5.1 which I imagine might just be caution around the cutting edge)  I was getting warnings-as-errors during the CI build for some code in this PR because the podspec version was out of date while my local version of Swift was up to date, so sneaking that version bump in here atomically to get the code to pass.